RNSquareInAppPayments.podspec

```diff
@@ -17,7 +17,7 @@ Pod::Spec.new do |s|
   s.resource_bundle = { "RNSquareInAppPayments-Resources" => ["ios/RNSquareInAppPayments-Resources/*.lproj/*.strings"] }
 
   s.dependency "React"
-  s.dependency "SquareInAppPaymentsSDK", '1.5.1'
-  s.dependency 'SquareBuyerVerificationSDK', '1.5.1'
+  s.dependency "SquareInAppPaymentsSDK", '1.4.0'
+  s.dependency 'SquareBuyerVerificationSDK', '1.3.0'
```

Comment:
Please use `1.4.0`, it was missing but it should work now.

Reply:
I tried changing SquareBuyerVerificationSDK (= 1.3.0) to SquareBuyerVerificationSDK (= 1.4.0) but the pod install fails with the following error.
[!] CocoaPods could not find compatible versions for pod "SquareBuyerVerificationSDK":
  In snapshot (Podfile.lock):
    SquareBuyerVerificationSDK (= 1.3.0)

  In Podfile:
    RNSquareInAppPayments (from `../node_modules/react-native-square-in-app-payments`) was resolved to 1.5.1, which depends on
      SquareBuyerVerificationSDK (= 1.4.0)

Specs satisfying the `SquareBuyerVerificationSDK (= 1.3.0), SquareBuyerVerificationSDK (= 1.4.0)` dependency were found, but they required a higher minimum deployment target.

I tried changing the deployment target to various version and the latest one but that wasn't successful either.
I also checked pod install with 1.5.1 which was successful but for that need to update both the SDK versions to 1.5.1 on iOS.
    - SquareBuyerVerificationSDK (= 1.5.1)
    - SquareInAppPaymentsSDK (= 1.5.1)